src/main/kotlin/com/hedvig/rapio/members/MembersController.kt

```diff
@@ -38,7 +40,7 @@ class MembersController(
         val trialEndDate = body.fromDate.plusDays(30)
         val today = LocalDate.now()
         require(!trialEndDate.isBefore(today)) {
-            return ResponseEntity.badRequest().build()
+            throw badRequest("Invalid fromDate: must not be more than 30 days ago")
```

Comment:
How come we throw a bad request? I'm 100% misunderstanding there just curious
```suggestion
            return badRequest("Invalid fromDate: must not be more than 30 days ago")
```

Reply:
We can't return it since it's not a `ResponseEntity`, and we can't use `ResponseEntity` because it doesn't allow error bodies